### PR TITLE
Fix/improvements

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
1. Fixed failed tests in WOPI Validator, made versions accumulation configurable.
2. Made Office Online editor frame in the full-page mode without topbar.
3. Fixed filename for downloading a file from the editor.
4. Changed WOPI Discovery Delay from 10 mins to 12h as recommended in docs.

### CODE REVIEW

File: services/src/main/java/org/exoplatform/officeonline/WOPIService.java [hunk 1]
```diff
@@ -425,7 +437,7 @@ public void putFile(EditorConfig config, String lockId, InputStream data) throws
 
         long timeout = System.currentTimeMillis() - versionDate.getTimeInMillis();
         // Version accumulation for same user
-        if (versionable && config.getUserId().equals(versioningUser) && timeout < VERSION_TIMEOUT) {
+        if (versionsAccumulation && versionable && config.getUserId().equals(versioningUser) && timeout < VERSION_TIMEOUT) {
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/WOPIService.java]: Add service methods:
```
isVersionAccumulationEnabled()
setVersionAccumulation(boolean)
```
And expose them to REST service scoped to admin users.

File: webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml [hunk 2]
```diff
@@ -54,6 +54,11 @@
         <property name="wopi-url"
           value="${officeonline.wopi.files.url:http://wopi-dev01.exoplatform.org/portal/rest/wopi}" />
       </properties-param>
+      <properties-param>
+        <name>versions-accumulation-configuration</name>
+        <property name="versions-accumulation"
+          value="${officeonline.versions.accumulation:false}" />
```
Comment (pnedonosko) [inline webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml]: Accumulation enabled by default = `true`.

**Review by pnedonosko — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml (hunk 2 above)
Comment (VladMartyshevskyi) [reply webapp/src/main/webapp/WEB-INF/conf/officeonline/configuration.xml]: Fixed: https://github.com/exo-addons/office-online/pull/32/commits/8273174fba8747732c8e179f333f5ede33d4c5b2

File: services/src/main/java/org/exoplatform/officeonline/WOPIService.java (hunk 1 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/WOPIService.java]: Updated: https://github.com/exo-addons/office-online/pull/32/commits/8273174fba8747732c8e179f333f5ede33d4c5b2

File: services/src/main/java/org/exoplatform/officeonline/rest/EditorResource.java [hunk 3]
```diff
@@ -81,35 +94,24 @@ public Response content(@Context UriInfo uriInfo,
...
-   * @param uriInfo the uri info
-   * @param request the request
-   * @param context the context
-   * @param userId the user id
-   * @param fileId the file id
-   * @return the response
-   */
   @GET
-  @Path("/test/token/{userId}/{fileId}")
+  @Path("/version/accumulation")
```
Comment (pnedonosko) [inline services/src/main/java/org/exoplatform/officeonline/rest/EditorResource.java]: add /configuration

**Review by pnedonosko — CHANGES_REQUESTED** [review]

File: services/src/main/java/org/exoplatform/officeonline/rest/EditorResource.java (hunk 3 above)
Comment (VladMartyshevskyi) [reply services/src/main/java/org/exoplatform/officeonline/rest/EditorResource.java]: Fixed: https://github.com/exo-addons/office-online/pull/32/commits/cea4b790466cc6ab9e3cd09d4b47f2d5eb8a006a

**Review by pnedonosko — APPROVED** [review]

**Pull request merged**